### PR TITLE
Add Help Page

### PR DESCRIPTION
**Pull request by gilleswittenberg**

### CODE REVIEW

File: src/app/features/help/routes.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import HelpPage from "./components/pages/Help/HelpPage"
+
+export default {
+  "/hulp": {
+    Page: HelpPage,
+    title: "Hulp",
+    icon: "Handicap"
```
Comment (remcohoff) [inline src/app/features/help/routes.tsx:7]: Handicap? :-) Wouldn't a questionmark be better?

File: src/app/features/shared/components/molecules/navigation/DefaultNavigation.tsx [hunk 2]
```diff
@@ -45,6 +45,9 @@ const DefaultNavigation: React.FC<Props> = ({ showSearchButton }) => {
       <ResponsiveMenuInline>
         <MenuItems />
       </ResponsiveMenuInline>
+      <ButtonLink to={to("/hulp")}>
+        <SearchButton size={50} variant="blank" iconSize={20} icon={<Handicap />} />
```
Comment (remcohoff) [inline src/app/features/shared/components/molecules/navigation/DefaultNavigation.tsx:49]: Maybe we should replace "SearchButton" with a more generic name, something like "IconButton"

**Review by remcohoff — APPROVED** [review]

File: src/app/features/shared/components/molecules/navigation/DefaultNavigation.tsx (hunk 2 above)
Comment (gilleswittenberg) [reply src/app/features/shared/components/molecules/navigation/DefaultNavigation.tsx:49]: Yes, will do

File: src/app/features/help/routes.tsx (hunk 1 above)
Comment (gilleswittenberg) [reply src/app/features/help/routes.tsx:7]: QuestionMark is not available from ASC. Have made a Jira task to discuss this.

**Pull request merged**